fireant/tests/widgets/test_reacttable.py

```diff
@@ -1050,6 +1050,59 @@ def test_dimx2_hide_dim1(self):
             result,
         )
 
+    def test_dimx2_metricx2_refx2_hide_metrics(self):
+        dimensions = [
+            day(mock_dataset.fields.timestamp),
+            mock_dataset.fields.political_party,
+        ]
+        references = [ElectionOverElection(mock_dataset.fields.timestamp)]
+        result = ReactTable(
+            mock_dataset.fields.votes,
+            mock_dataset.fields.wins,
+            hide=[mock_dataset.fields.votes, mock_dataset.fields.wins],
+        ).transform(dimx2_date_str_ref_df, dimensions, references)
+
+        self.assertIn("data", result)
+        result["data"] = result["data"][:2]  # shorten the results to make the test easier to read
```

Comment:
Can you hide a reference? Not sure if that would ever be useful though.

Reply:
Nevermind, I see the use case - hiding a specific metric reference. Looks like this is tested in the pandas transformer anyway.

Reply:
That's useful when you have multiple metrics and yet is only interested in the reference of a subset of them.